roles/transformers/vars/main.yml

```diff
@@ -9,3 +9,5 @@ libreoffice_home: "/opt/libreoffice{{ dependencies_version.libreoffice[0:3] }}/"
 
 alfresco_pdf_renderer: "{{ nexus_repository.releases }}/alfresco-pdf-renderer/{{ dependencies_version.pdf_renderer }}/alfresco-pdf-renderer-{{ dependencies_version.pdf_renderer }}-linux.tgz"
 alfresco_pdf_renderer_sha1_checksum_url: "{{ alfresco_pdf_renderer }}.sha1"
+
+imagemagick_repo: "https://github.com/Alfresco/imagemagick-build/releases/download"
```

Comment:
```suggestion
imagemagick_repo: "https://github.com/Alfresco/imagemagick-build/releases/download"

```

Reply:
Although github sounds like a better place to store than nexus I think we should still store artifact in one single place to avoid having too many dependencies.
Built packages should be uploaded to and gathered from nexus.
Also I think there are some restrictions to ImageMagick distribution which I'm not sure we conform to on this repo. At least this should be done with caution and with legal approval (https://imagemagick.org/script/license.php).
Generally speaking moving artifacts to a different place should be done in a consolidated way as part of a different ticket